src/instance.cpp

```diff
@@ -3,10 +3,12 @@
 #include <capo/sound.hpp>
 #include <capo/source.hpp>
 #include <impl_al.hpp>
+#include <cassert>
```

Comment:
Where is assert used?

Reply:
It's used in my latest changes, but here it's unused.